modules/search/instant-search/components/search-result-product.jsx

```diff
@@ -0,0 +1,62 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h, Component } from 'preact';
+
+/**
+ * Internal dependencies
+ */
+import SearchResultComments from './search-result-comments';
+
+class SearchResultProduct extends Component {
+	render() {
+		const { result_type, fields, highlight } = this.props.result;
+		if ( result_type !== 'post' ) {
+			return null;
+		}
+
+		const firstImage = Array.isArray( fields[ 'image.url.raw' ] )
+			? fields[ 'image.url.raw' ][ 0 ]
+			: fields[ 'image.url.raw' ];
+
+		return (
+			<li className="jetpack-instant-search__result-product">
+				<h3>
+					<a
+						href={ `//${ fields[ 'permalink.url.raw' ] }` }
+						target="_blank"
+						rel="noopener noreferrer"
+						className="jetpack-instant-search__result-product-title"
+						//eslint-disable-next-line react/no-danger
+						dangerouslySetInnerHTML={ { __html: highlight.title } }
+					/>
+				</h3>
+				{ firstImage && (
+					<img
+						className="jetpack-instant-search__result-product-img"
+						src={ `//${ firstImage }` }
+						alt=""
+					/>
+				) }
+				<div
+					className="jetpack-instant-search__result-product-content"
+					//eslint-disable-next-line react/no-danger
+					dangerouslySetInnerHTML={ {
+						__html: highlight.content.join( ' ... ' ),
+					} }
+				/>
+				{ fields[ 'wc.price' ] && (
+					<div className="jetpack-instant-search__result-product-price">
+						{ fields[ 'wc.price' ].toFixed( 2 ) }
```

Comment:
Does this normalize correctly for other currencies?

Reply:
Not yet - the API doesn't yet provide the currency (cc @gibrown).

We could potentially use `getCurrencyDefaults` from `@automattic/format-currency` (this is what the Simple Payments button uses in Calypso.

I'll add an issue for this.

Reply:
FYI, the field we are taking this from is a float, so I think this will always work correctly.

Reply:
I think it's to address the fact that, although most currencies use 2 decimal places, there are a number that use no decimal places (like Japanese yen) or 3 decimal places (like the Libyan dinar): https://stackoverflow.com/questions/2701301/various-countrys-currency-decimal-places-width-in-the-iphone-sdk